assets/css/index.css

```
@@ -1340,25 +1340,41 @@ input[type="range"]::-webkit-slider-thumb:hover {
+	padding: 2px 4px;
 	color: #10b981;
 	font-weight: 500;
 }
+
 .audio-indicator svg {
-	width: 20px;
-	height: 20px;
+	width: 18px;
+	height: 18px;
+	flex-shrink: 0;
 	fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)